packages/homebrew/layer0.rb

```diff
@@ -0,0 +1,22 @@
+class Layer0 < Formula
+  desc "Framework that helps you deploy to the AWS with minimal fuss"
+  homepage "http://layer0.ims.io"
+  url "https://github.com/quintilesims/layer0/archive/v0.10.6.tar.gz"
```

Comment:
this is targeting v0.10.X releases, but the PR is targeting `api-refactor` --- we should either preemptively switch to a v0.11 release that doesn't exist yet, or target this PR for `develop`.

Reply:
Yup, I'll go ahead and change that.

Reply:
Actually I think I'll leave it until 11 is read so we can at least test this.